examples/lima/coder.yaml

```diff
@@ -93,10 +84,8 @@ provision:
       # Ensure coder starts on boot
       systemctl enable coder
       systemctl start coder
-      # Wait for Coder to have downloaded Terraform
-      timeout 60s bash -c 'until /var/cache/coder/terraform version >/dev/null 2>&1; do sleep 1; done'
-      # Coder restarts after downloading Terraform, wait for it to become available
-      timeout 60s bash -c 'until nc -z localhost 3000 > /dev/null 2>&1; do sleep 1; done'
+      # Wait for Terraform to be installed
+      timeout 60s bash -c 'until /usr/local/bin/terraform version >/dev/null 2>&1; do sleep 1; done'
```

Comment:
review: terraform path updated

Reply:
My thinking here is that if terraform installation fails for some reason, it will be more obvious to the end-user.